### PR TITLE
menu stuff

### PR DESCRIPTION
**Pull request by jonsk197**

### CODE REVIEW

File: src/Entry.h [hunk 1]
```diff
@@ -1,14 +1,20 @@
-//
-//  Entry.h
-//  LeapMusic
-//
-//  Created by Jonas Skog Andersen on 2015-02-06.
-//
-//
+#ifndef Entry_H
+#define Entry_H
```
Comment (Rovanion) [inline src/Entry.h]: Becaus Entry_H is such a generic name, could I ask that this header guard is named LeapMusic_Entry_h?

File: src/Entry.h [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+#include "../include/Leap.h"
+#include <string>
 
-#ifndef __LeapMusic__Entry__
-#define __LeapMusic__Entry__
 
-#include <stdio.h>
+using namespace Leap;
```
Comment (Rovanion) [inline src/Entry.h:7]: Why are you using the Leap namespace?

File: src/Menu.h [hunk 3]
```diff
@@ -2,19 +2,21 @@
 #ifndef MENU
 #define MENU
```
Comment (Rovanion) [inline src/Menu.h]: This should also be named LeapMusic_Menu

File: src/Entry.h (hunk 1 above)
Comment (jonsk197) [inline src/Entry.h]: Yes!

File: src/Entry.h (hunk 2 above)
Comment (jonsk197) [inline src/Entry.h:7]: Dont know, there since old times.

File: src/Menu.h (hunk 3 above)
Comment (jonsk197) [inline src/Menu.h]: ok

File: src/Entry.h (hunk 2 above)
Comment (Rovanion) [inline src/Entry.h:7]: Remove it if you aren't using a LOT of LeapMotion stuff in the class.

**Pull request merged**